dogpile/cache/api.py

```diff
@@ -190,7 +195,7 @@ def __init__(self, arguments: BackendArguments):  # pragma NO COVERAGE
         raise NotImplementedError()
 
     @classmethod
-    def from_config_dict(cls, config_dict, prefix):
+    def from_config_dict(cls, config_dict: Mapping[str, Any], prefix: str) -> Self:
```

Comment:
Adding types here showed something interesting:

https://github.com/sqlalchemy/dogpile.cache/blob/fd9d31a72f9b8bef090f9496eacdfc77fc2ea0da/dogpile/cache/region.py#L497-L500

`_config_prefix` can be `None` according to the `__init__` signature, but `from_config_dict` requires a non-`None` `str` value

Reply:
it's assumed they would go together as both None or both present.   these are private variables so if typing is failing use `type: ignore` for now